setup.py

```diff
@@ -1,13 +1,12 @@
-from setuptools import setup, find_packages
-import sys
+from setuptools import find_packages, setup
 
 setup(
-    name='lcls-tools',
-    version='0.1.dev0',
```

Comment:
Version was removed?

Reply:
Ah fuck that was when I was trying to fix the reinstallation. I'll put it back

Reply:
ty!